### PR TITLE
monitor for possible abuse reporting fails and email alerts

### PR DESCRIPTION
**Pull request by eviljeff**
WIP for #21769 - based on the initial proposal to email people directly, which is now probably not what we want...

### CODE REVIEW

File: src/olympia/abuse/cron.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    """check for any abuse reports created more than Y business days ago that have a
+    cinder job ID but don’t have a cinder decision."""
+    sla_date = datetime.now() - timedelta(days=3)
+    return (
+        CinderJob.objects.unresolved()
+        .exclude(
+            id__in=CinderJob.objects.resolvable_in_reviewer_tools().values_list(
+                'id', flat=True
+            )
+        )
```
Comment (eviljeff) [inline src/olympia/abuse/cron.py:31]: there's a more efficient way to do this, but it involves rewriting `resolvable_in_reviewer_tools` to use `Q`s and being able to pass a negation arg to get cases that _aren't_ resolvable in the reviewer tools, i.e. are resolvable in cinder instead.

File: src/olympia/abuse/cron.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+from olympia.amo.utils import send_mail
+
+from .models import AbuseReport, CinderJob
+
+
+def reports_without_cinder_id_qs():
+    """check for any abuse reports that have been created more than an hour ago and
+    don’t have a cinder job yet"""
+    one_hour_ago = datetime.now() - timedelta(hours=1)
```
Comment (eviljeff) [inline src/olympia/abuse/cron.py:13]: these should be at least defined as constants. (and etc in this file)

File: src/olympia/abuse/cron.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        .resolvable_in_reviewer_tools()
+        .filter(created__lt=sla_date)
+    )
+
+
+def abuse_report_health_checks():
+    for subject, to, template_file, qs in (
+        (
+            '%s abuse reports without a cinder_job',
+            ['andreas@'],
```
Comment (eviljeff) [inline src/olympia/abuse/cron.py:51]: these email addresses should be defined in Config objects.

**Comment by diox** [comment]
Closing in favor of https://github.com/mozilla/addons-server/pull/21805 + doing the alerting in redash.

**Pull request closed without merging**